src/main/java/techcourse/myblog/domain/comment/CommentRepository.java

```diff
@@ -0,0 +1,10 @@
+package techcourse.myblog.domain.comment;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface CommentRepository extends JpaRepository<Comment, Long> {
+    Optional<List<Comment>> findByArticleArticleId(Long articleId);
```

Comment:
`List<Comment>`를 `Optional`로 포장할 필요가 있을까요? 메서드 이름도 `findByArticleArticleId()` 보다는 `findAllByArticleArticleId()`가 명확해 보여요.

```suggestion
    List<Comment> findAllByArticleArticleId(Long articleId);
```